src/db_handler.py

```diff
@@ -1,10 +1,11 @@
-from sqlalchemy import *
-from sqlalchemy.orm import sessionmaker
-from sqlalchemy.ext.automap import automap_base
-from sqlalchemy import exc
 import logging
 from logging.handlers import RotatingFileHandler
 
+from sqlalchemy import *
```

Comment:
Adding * isn't great, because you can't just find where function or variable came from and you should google it or have a good IDE to find out if it's from some module, or from within a project or smth else